### ISSUE TITLE
Ultron Website is not accessible

### ISSUE THREAD

**Issue by contactsinghal** [UltronFoundationDev/Mainnet#1]
As a Accessibility tester I have found issues in the "Ultron.foundation", "Lottoday.io" websites. As per the accessibility laws websites should be compliant to accessibility. Otherwise, heavy penalties can be put by the respective Government as per the laws in their regions.
For more details and reference, please find the attached PPT.

[Accessibility-in-Web.pptx] (https://github.com/UltronFoundationDev/Mainnet/files/14211074/Accessibility-in-Web.pptx)

Contact Details:
Mohit Singhal
Mobile: +91 9999709511
Email: contactsinghal@gmail.com

**Comment by contactsinghal** [comment]
MySelf certified accessibility tester from Homeland Security Services USA. I have more than 6 years of experience in Accessibility testing. Attached is the certificate for reference.
[Trusted Tester for Web  Certificate Mohit Singhal.pdf](https://github.com/UltronFoundationDev/Mainnet/files/14211175/Trusted.Tester.for.Web.Certificate.Mohit.Singhal.pdf)
